READ_BEFORE_UPDATE.md

```diff
@@ -2,5 +2,15 @@
 
 ## Update from 2.21 to 2.22
 
+* Some combinations of installed packages may cause problems with the LDAP resolver:
+  * ldap3 2.1.1, pyasn1 0.4.2: Attempts to use STARTTLS fail with an exception (Issue #885)
+  * ldap3 >= 2.3: The user list appears to be empty, the log shows LDAPSizeLimitExceededResult exceptions. (Issue #887)
+                  This can be mitigated by setting a size limit greater than the number of users in the LDAP directory.
+  * ldap3 >= 2.4: User attributes are not retrieved properly (Issue #911)
+  * ldap3 >= 2.4.1: UnicodeError on authentication, token view displays resolver errors (Issue #911)
+  * ldap3 >= 2.4.1: Cannot search for non-ASCII characters in user view (#980)
```

Comment:
What is the difference between "ldap3 >= 2.4" and "ldap3 >= 2.4.1".

Should we point out, that the pip installation will install the combination of 2.1.1/0.4.2 and the ubuntu installation will install 2.1.1/0.1.9?

Reply:
Well, the UnicodeError only occurs in 2.4.1, not 2.4 :-) Maybe this is a confusing way to write that.

Good idea, I'll add a note!